docs/source/demos/text_input.ipynb

```diff
@@ -240,7 +239,7 @@
    "source": [
     "## Why encode text this way?\n",
     "\n",
-    "To demonstrate the importance of text-specific modeling, let's train a model with the same dataset, without letting `AutoMLSearch` detect the text column. We can change this by explicitly setting the data type of the `'Message'` column in Woodwork."
+    "To demonstrate the importance of text-specific modeling, let's train a model with the same dataset, without letting `AutoMLSearch` detect the text column. We can change this by explicitly setting the data type of the `'Message'` column in Woodwork to `Categorical`."
```

Comment:
Nit pick but should "categorical" be lowercase? Maybe case doesn't matter for woodwork type matching, idk

Reply:
Ah I've always just used "Categorical" since that's what the [actual name](https://woodwork.alteryx.com/en/stable/_modules/woodwork/logical_types.html#Categorical) is but you're right, woodwork type matching doesn't matter 🤷